pydropsonde/circles.py

```diff
@@ -176,7 +176,7 @@ def add_circle_variables_to_ds(self):
                 ),
                 circle_time=(
                     [],
-                    self.circle_ds["sonde_time"].mean().values,
+                    self.circle_ds["sonde_time"].isel(sonde=[0, -1]).mean().values,
```

Comment:
thanks for adjusting the circle "reference" time! I think that this is the better solution compared to a simple mean. Could you also adjust the `circle_time["description"]` in line 168 to reflect the change, e.g. "Mean launch time of first and last sonde in circle"?